python/src/mas/cli/uninstall/app.py

```diff
@@ -184,7 +185,8 @@ def uninstall(self, argv):
                     uninstallCommonServices = uninstallCommonServices,
                     uninstallUDS = uninstallUDS,
                     uninstallMongoDb = uninstallMongoDb,
-                    uninstallSLS = uninstallSLS
+                    uninstallSLS = uninstallSLS,
+                    droNamespace = droNamespace
```

Comment:
Doesn't it has to follow the python-devops signature? 
![image](https://github.com/user-attachments/assets/02800363-fac2-4458-aba2-5b1cdd064e5b)

Reply:
fixed